README.md

```diff
@@ -45,15 +52,15 @@ Uri                | GET | POST | desc
 /common/set_remote_method | | X | Set information on the polling system (reduce remote time update ??)
 /aircon/get_model_info | X | | Provides model informarion
 /aircon/get_control_info | X | | Main Uri to request all current status parameters
-/aircon/set_control_info | | X | Main Uri to set status parameters ( control almost all)
+/aircon/set_control_info | X | X | Main Uri to set status parameters (control almost all)
```

Comment:
Hello,

pi@dev:~ $ curl -i "http://192.168.0.5/aircon/set_control_info?pow=1&mode=3&stemp=24.0&shum=0&f_rate=A&f_dir=0"
HTTP/1.0 200 OK
Content-Length: 11
Content-Type: text/plain

ret=OK,adv=

It works perfectly. There is an identical response if you call URL through web browser.